pyaedt/generic/pdf.py

```diff
@@ -104,53 +256,91 @@ def aedt_version(self):
         -------
         str
         """
-        return self.report_specs.AnsysVersion
+        return self.report_specs.ansys_version
 
     @aedt_version.setter
     def aedt_version(self, value):
-        self.report_specs.AnsysVersion = value
+        self.report_specs.ansys_version = value
 
-    def add_section(self, portrait=True):
+    def add_section(self, portrait=True, page_format="a4"):
         """Add a new section to Pdf.
 
         Parameters
         ----------
         portrait : bool, optional
             Section orientation. Default ``True`` for portrait.
+        page_format : str, optional
+            Currently supported formats are a3, a4, a5, letter, legal or a tuple (width, height).
 
         Returns
         -------
         int,
             Section id.
         """
-        orientation = "P" if portrait else "L"
-        return self.report.CreateNewSection(orientation)
+        orientation = "portrait"
+        if not portrait:
+            orientation = "landscape"
+        self.add_page(orientation=orientation, format=page_format)
 
-    def add_chapter(self, chapter_name, section_id=0):
+    def add_chapter(self, chapter_name):
         """Add a new chapter.
 
         Parameters
         ----------
         chapter_name : str
             Chapter name.
-        section_id : int, optional
-            Section on which add the chapter. Default is 0 which use current section.
         """
-        self.report.AddChapter(chapter_name, section_id)
+        self._chapter_idx += 1
+        self._sub_chapter_idx = 0
+        txt = f"{self._chapter_idx} {chapter_name}"
```

Comment:
```suggestion
        txt = f"{self.__chapter_idx} {chapter_name}"
```

Reply:
@maxcapodi78 (warning) here you are using f-strings and I don't think that they are available in IronPython (or at least I coudn't find them), if this feature isn't intended to be used in IronPython, I would advise you follow my first comment an use dataclasses for the top class of this file.

Reply:
this has not to be compatible with IPY because fpdf is not supported by IPY